lib/refresh.js

```diff
@@ -95,7 +95,7 @@ issueQueue.pop(function(githubIssue, next) {
    gitManager.parseIssue(githubIssue)
    .then(dbManager.updateAllIssueData)
    .done(function() {
-      debug("done refreshing issue %s", githubIssue.number);
+      debug("done refreshing issue %s in repo %s", githubIssue.number, githubIssue.repo);
```

Comment:
Does `githubIssue` have a `repo` property?

Reply:
I'd hope so, git manager calls parse issue above which accesses it.